source/index.html.md.erb

```diff
@@ -15852,48 +15897,70 @@ HTTP status code | Smartsheet errorCode | Smartsheet message
 
 # Formatting
 
-Formatting data can optionally be included for columns, rows, and cells by adding the "include=format" query string parameter to any API operation that returns any of those objects (for example, `GET /sheets/{sheetId}`, `GET sheets/{sheetId}/rows/{rowId}`, etc.).  When this parameter is included, objects that contain non-default formatting include a **format** property.  Objects that do not have any non-default format settings applied exclude this property.
- 
-Because the amount of format data in a large sheet can potentially be very large in itself, for bandwidth reasons, format data is represented by a compact format descriptor string.  The format descriptor takes the form of a comma-separated list of numeric values.  These values represent indexes into pre-defined format lookup tables which can be retrieved using the `GET /serverinfo` operation.
+> Example defaults value:
 
-## Row & Column Format
+```json
+{
+	"defaults": "0,2,0,0,0,0,0,0,0,0,0,0,0,0,0,0"
+}
+```
 
-Setting the format of a row or column through the API simply sets the baseline format for new cells in that row or column.  It does not affect existing cells.
- 
-The equivalent API action for what happens when you highlight a column or row in the Smartsheet Webapp and set a format (for example, bold) is to update the format of a column or row (via `PUT /sheets/{sheetId}/rows/{rowId}` or `PUT /sheets/{sheetId}/columns/{columnId}`), and update the format for every cell in that row or column.  All of the cells in a row can be updated in one operation (`PUT /sheets/{sheetId}/rows/{rowId}`), but there is currently no way to update all of the cells in a column in one operation.
+To set or read formatting programmatically, Smartsheet uses a compact format string, **cell.format**, which looks something like this: ",,1,1,,,,,,,,,,,,". The position and sample values in this string are explained in the following format descriptor table:
 
-## Format Descriptor
+## Format Descriptor Table
 
-The format descriptor contains 16 comma-separated numeric indexes.  Each index represents a format style whose value can be found in one of the lookup tables in the [FormatTables Object](#formattables-object).
+**Position** | **Lookup Property** | **Example Value** | **Model**
+-----|-----|-----|-----|
+0 | fontFamily | 0 = Arial, default | "0,,,,,,,,,,,,,,,"
```

Comment:
I like this table.
Can we get the samples in a monospaced (code) font? 
I think the Examples should match the Models
Rather than 'Model' - how about 'format string'?

Reply:
Done.